app/api/ada/index.js

```diff
@@ -557,9 +559,45 @@ export default class AdaApi {
       }
     }
   }
+
+  // noinspection JSMethodCanBeStatic
+  async getTransactionRowsToExport(): Promise<Array<TransactionExportRow>> {
+    try {
+      await refreshTxs();
+      const history: AdaTransactions = await getAdaTxsHistoryByWallet();
+      return AdaApi.convertAdaTransactionsToExportRows(history[0]);
+    } catch (e) {
+      Logger.error('AdaApi::exportTransactionsToFile: ' + stringifyError(e));
+      throw e;
+    }
+  }
+
+  static convertAdaTransactionsToExportRows(
+    transactions: Array<AdaTransaction>
+  ): Array<TransactionExportRow> {
+    return transactions
+      .filter(tx => tx.ctCondition === 'CPtxInBlocks')
+      .map(tx => {
+        const fullValue = parseInt(tx.ctAmount.getCCoin, 10);
```

Comment:
I think this overflows, so probably we should use `BigInt`

Reply:
Oh that's a really good catch, yeah. Replaced with `BigNumber` 👌